src/main/scala/subsystem/BaseSubsystem.scala

```diff
@@ -36,6 +36,14 @@ abstract class BareSubsystemModuleImp[+L <: BareSubsystem](_outer: L) extends La
   println(outer.dts)
 }
 
+
+sealed trait BaseSubsystemBusAttachment
+case object SBUS extends BaseSubsystemBusAttachment
+case object PBUS extends BaseSubsystemBusAttachment
+case object FBUS extends BaseSubsystemBusAttachment
+case object MBUS extends BaseSubsystemBusAttachment
+case object CBUS extends BaseSubsystemBusAttachment
```

Comment:
@rmac-sifive Imagine these as `case class CBUS(idx: Int)` ...

Reply:
So then somewhere in my config `Key`s for the IP I set attachment point  `= CBUS(0)` or 
`= CBUS(1)` depending on which subsystem I want the IP to attach to? Makes sense, and then we do something like:

Reply:
Yes that is lines i was thinking along!